Tests/SRGDataProviderNetworkTests/ContentServicesTestCase.m

```diff
@@ -118,6 +118,59 @@ - (void)testContentPageForTopic
     [self waitForExpectationsWithTimeout:30. handler:nil];
 }
 
+- (void)testContentPageForShow
+{
+    XCTestExpectation *expectation1 = [self expectationWithDescription:@"Request succeeded"];
+    
+    [[self.dataProvider contentPageForVendor:SRGVendorSRF product:SRGProductPlayVideo showWithURN:@"urn:srf:show:tv:ff969c14-c5a7-44ab-ab72-14d4c9e427a9" published:YES atDate:nil withCompletionBlock:^(SRGContentPage * _Nullable contentPage, NSHTTPURLResponse * _Nullable HTTPResponse, NSError * _Nullable error) {
+        XCTAssertNotNil(contentPage);
+        XCTAssertNil(error);
+        [expectation1 fulfill];
+    }] resume];
+    
+    [self waitForExpectationsWithTimeout:30. handler:nil];
+    
+    XCTestExpectation *expectation2 = [self expectationWithDescription:@"Request succeeded"];
+    
+    [[self.dataProvider contentPageForVendor:SRGVendorSRF productName:@"PLAY_VIDEO" showWithURN:@"urn:srf:show:tv:ff969c14-c5a7-44ab-ab72-14d4c9e427a9" published:YES atDate:nil withCompletionBlock:^(SRGContentPage * _Nullable contentPage, NSHTTPURLResponse * _Nullable HTTPResponse, NSError * _Nullable error) {
+        XCTAssertNotNil(contentPage);
+        XCTAssertNil(error);
+        [expectation2 fulfill];
+    }] resume];
+    
+    [self waitForExpectationsWithTimeout:30. handler:nil];
+    
+    XCTestExpectation *expectation3 = [self expectationWithDescription:@"Request succeeded"];
```

Comment:
Is it really useful to test `SRGProductPlayVideo`  and `PLAY_VIDEO`?

Reply:
As the two methods exist, I would say yes.